zebra/zebra_vxlan.c

```diff
@@ -5913,6 +5915,7 @@ void zebra_vxlan_close_tables(struct zebra_vrf *zvrf)
 		return;
 	hash_iterate(zvrf->evpn_table, zebra_evpn_vxlan_cleanup_all, zvrf);
 	hash_free(zvrf->evpn_table);
+	zvrf->evpn_table = NULL;
```

Comment:
Hi, why this pull request was closed? I seem to have encountered the same problem